pymc3/distributions/continuous.py

```diff
@@ -298,7 +298,7 @@ def __init__(self, sd=None, tau=None, *args, **kwargs):
         assert_negative_support(sd, 'sd', 'HalfNormal')
 
     def random(self, point=None, size=None, repeat=None):
-        sd = draw_values([self.sd], point=point)
+        sd = draw_values([self.sd], point=point)[0]
```

Comment:
`draw_values` used to return a list if it was passed more than one value and a single value otherwise. I didn't like that and change it to return always a list. I am playing around with `sample_ppc` a bit, and there it could be that the list of variable to draw is dynamic, which would make the previous behaviour somewhat strange. But I can change it back if you disagree.